packages/app/src/debug/DebugRunNavigationLimitMessage.vue

```diff
@@ -22,9 +22,12 @@
 import ExternalLink from '@cy/gql-components/ExternalLink.vue'
 import { useI18n } from '@cy/i18n'
 import { IconWarningCircle } from '@cypress-design/vue-icon'
+import { getUrlWithParams } from '@packages/frontend-shared/src/utils/getUrlWithParams'
 
 const { t } = useI18n()
 
-defineProps<{ cloudProjectUrl?: string }>()
+const props = defineProps<{ cloudProjectUrl?: string }>()
+
+const cloudProjectUrlWithUtmParams = props.cloudProjectUrl && getUrlWithParams({ url: props.cloudProjectUrl, params: { utm_medium: 'Debug Tab', utm_campaign: 'Run Navigation Limit' } })
```

Comment:
Should this be `computed(() => {})` so it's reactive to prop changes after mount?

Reply:
Done